recipes/muparserx/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class MuparserxConan(ConanFile):
+    name = "muparserx"
+    description = "Math Parser Library"
+    license = "BSD"
+    topics = ("conan", "muparserx", "math", "parser")
+    homepage = "https://beltoforion.de/article.php?a=muparserx"
+    url = "https://github.com/beltoforion/muparserx"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": False}
+    generators = "cmake"
+
```

Comment:
Add caching of cmake to avoid reconfiguration between build and package (part 1/2)
```suggestion

    _cmake = None

```